packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -21,11 +21,15 @@ interface ScriptEditorProps {
   focusOnMount?: boolean;
   onChange: (e: editor.IModelContentChangedEvent) => void;
   onRunCommand: (command: string) => void;
+  onEditorInitialized: (editor: editor.IStandaloneCodeEditor) => void;
+  onEditorWillDestroy: () => void;
```

Comment:
`Editor` passes through the editor as a param, we should here as well:
```suggestion
  onEditorWillDestroy: (editor: monaco.editor.IStandaloneCodeEditor) => void;
```